### ISSUE TITLE
the priority of coc_user_config is higher than workspace

### ISSUE THREAD

**Issue by pzxbc** [neoclide/coc.nvim#1403]
## Result from CocInfo

```
## versions

vim version: NVIM v0.4.3
node version: v13.3.0
coc.nvim version: 0.0.74-d83536aa4e
term: xterm-256color
platform: linux

## Messages
E464: Ambiguous use of user-defined command
## Output channel: explorer


## Output channel: prettier


## Output channel: highlight
```

[Info  - 2:00:00 PM] Highlight server running in node v13.3.0

## Describe the bug

The priority of coc_user_config is higher than workspace. workspace config have the highest priority in usually

## Reproduce the bug

**We will close your issue when you don't provide minimal vimrc and we can't
reproduce it**

- Create file `mini.vim` with：

  ```vim
  set nocompatible
  set runtimepath^=/path/to/coc.nvim
  filetype plugin indent on
  syntax on
  set hidden
  let g:coc_user_config = {                      
            \'explorer.icon.enableNerdfont': v:true
            \}
  ```

- Start (neo)vim with command: `vim -u mini.vim`

- install coc.explorer with command `:CocInstall coc-explorer`

- OpenVim in some directory and toggle coc-explorer with command `:CocCommand explorer<CR>`, you wiil see nerd icon or a ? in the beginning of file name or directory.

- set workspace config

```
:CocLocalConfig
```

`coc-settings.json`

```
{
  "explorer.icon.enableNerdfont":false
}
```

- after setting local config. the nerd font icon is still active

**Comment by Gee19** [comment]
Seems to be fixed  by e5b05bee4ab3ccd959a275972284e85dcb296b5a

:)

**Comment by chemzqm** [comment]
@Gee19 No.
It's expected, so user can use viml to change configuration of coc.nvim.
